src/entity/EnemyShipSet.java

```diff
@@ -0,0 +1,151 @@
+package entity;
+
+import java.util.*;
+
+import screen.Screen;
+import engine.Cooldown;
+import engine.Core;
+import engine.DrawManager;
+import engine.DrawManager.SpriteType;
+import engine.GameSettings;
+import java.util.logging.Logger;
+
+public class EnemyShipSet {
+
+    // 생성된 적들을 관리하기 위한 Set
+    private Set<EnemyShip> enemies;
+    // 적 생성 쿨타임
+    private Cooldown spawnCooldown;
+    // DrawManager 인스턴스
+    private DrawManager drawManager;
+    // 게임 화면 정보
+    private Screen screen;
+    // 적 랜덤 생성을 위한 랜덤 객체
+    private Random random;
+    // 아군 함선 참조를 위한 변수
+    private Ship ship;
+    // 적 함선의 이동속도
+    private int movementSpeed;
+    // 적 함선의 X 방향 속도
+    private double X_SPEED = 1.0;
+    // 적 함선의 Y 방향 속도
+    private double Y_SPEED = 1.0;
```

Comment:
승인했습니다!

- 제가 코드 제안한 거에 X_SPEED 처럼 SPEED가 대문자로 되어있는데 실제 필드를 사용하는 곳에는 X_speed 로 되어있어서 컴파일 오류가 나네요. 이부분만 확인 부탁드립니다.

```suggestion
    // 적 함선의 X 방향 속도
    private double X_speed = 1.0;
    // 적 함선의 Y 방향 속도
    private double Y_speed = 1.0;
```